### PR TITLE
chore: remove decentraland-rpc and publish scene runtime systems

### PR DESCRIPTION
**Pull request by menduz**
# What? 
- Remove `decentraland-rpc` from this repository, it was not part of the CI and it was not included as a direct dependency in any project. We will keep the original repository at [decentraland/decentraland-rpc](https://github.com/decentraland/decentraland-rpc)
- Publish `cli.scene.system.js` and `scene.system.js` in the package `decentraland-ecs`, those will be used by the node debuggers.

# Why? 
Housekeeping and unlocking remote debugger.

### CODE REVIEW

**Review by nchamo — APPROVED** [review]
Looks good!

Could you link to the new repo where decentraland-rpc will be? Only for documentation purposes

**Comment by menduz** [comment]
Added to the PR description

File: kernel/.prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pablitar) [inline kernel/.prettierrc.json:1]: Why is this file added? AFAIK we are configuring prettier through `package.json`: https://github.com/decentraland/explorer/blob/master/kernel/package.json#L10

Is this better somehow?

**Review by pablitar — APPROVED** [review]
LGTM! Please coordinate with @sandrade-dcl before merging this to master, to ensure this doesn't conflict with the release!

File: kernel/.prettierrc.json (hunk 1 above)
Comment (menduz) [reply kernel/.prettierrc.json:1]: I had problems with VSCode while formatting and opening the explorer folder (it works well opening explorer/kernel tho). I know @BrianAmadori also had a similar issue.
